spec/index.bs

```diff
@@ -3926,6 +4029,48 @@ dictionary GPURenderPassDepthStencilAttachmentDescriptor {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPURenderPassDepthStencilAttachmentDescriptor>
+    : <dfn>attachment</dfn>
+    ::
+        A {{GPUTextureView}} describing the texture [=subresource=] the [=pipeline=] will output to
+        and read from for this depth/stencil attachment.
+
+    : <dfn>depthLoadValue</dfn>
+    ::
+        Issue: Describe this dictionary member
+
+    : <dfn>depthStoreOp</dfn>
+    ::
+        Issue: Describe this dictionary member
+
+    : <dfn>depthReadOnly</dfn>
```

Comment:
I wonder if we might need to require that `depthReadOnly` implies `depthLoadOp == "load"`, likewise for stencil.

Reply:
since the load ops are required, they can't currently be implied
but yes, we do need some wording to ensure that `depthReadOnly == true` can only be used with "load".

Reply:
I think depthReadOnly also means storeOp needs to be "store".

Reply:
both store ops don't make any sense for the read-only depth

Reply:
I think depthStoreOp "store" makes sense because the pass-local depth values wouldn't be able to change at all with depthReadOnly. So you "load" the original value and "store" without any changes.

Reply:
When you do this, you don't actually "store" anything. I.e. the very semantics a StoreOp defines doesn't work here. It may work "as-if" you are storing. But then - I can say that "discard" is "as-if" you are storing also. The whole point of "StoreOp" is optimization, hence "as-if" isn't applicable.

Reply:
Isn't "discard" as-if you are storing zero? That would be semantically different.

Reply:
Added the "load" requirement if either of the `readOnly`s are `true`. I'll let you talk amongst yourselves to figure out how the `storeOp`s figure into it.

Reply:
@kainino0x ok, that wasn't the best example. I'm not a fan of "store" for RO depth, but I don't see a good alternative right now (don't want to introduce Undefined or another variant of StoreOp just for that). So we can agree on "store" for now.

Reply:
Does the depth/stencil attachment need to have `OUTPUT_ATTACHMENT` usage if both `readOnly`s are false?

Reply:
yes it does

Reply:
It needs OUTPUT_ATTACHMENT regardless of any of the flags here, even if you can't actually output to it at all.

Reply:
@kvark 
```webidl
dictionary GPURenderPassDepthStencilAttachmentDescriptor {
    required GPUTextureView attachment;
    required (GPURenderPassDepthStencilReadOnly or GPURenderPassDepthReadWriteDescriptor) depth;
    required (GPURenderPassDepthStencilReadOnly or GPURenderPassStencilReadWriteDescriptor) stencil;
};
enum GPURenderPassDepthStencilReadOnly {
    "readonly"
};
dictionary GPURenderPassDepthReadWriteDescriptor {
    required (GPULoadOp or float) loadValue;
    required GPUStoreOp storeOp;
};
dictionary GPURenderPassStencilReadWriteDescriptor {
    required (GPULoadOp or GPUStencilValue) loadValue;
    required GPUStoreOp storeOp;
};
```
maybe?

Reply:
that's fancy! Do you think we could merge those structs? i.e. having something like `GPULoadOp or number`, where number can dynamically be float or int

Reply:
It's certainly possible, we can just say "convert the number to GPUStencilValue" in text or something. But I would prefer to leave it this way and let IDL take care of that. Plus it couldn't be reused in GPURenderPassColorAttachmentDescriptor (except using a union/any) since the type there is totally different.